src/api/pulls.rs

```diff
@@ -239,6 +239,36 @@ impl<'octo> PullRequestHandler<'octo> {
         self.http_get(url, None::<&()>).await
     }
 
+    /// Request a review from users or teams.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// use octocrab::params;
+    /// let review = octocrab::instance().pulls("owner", "repo")
+    ///    .request_reviews(101, vec!["user1".to_string(), "user2".to_string()], vec!["team1".to_string(), "team2".to_string()])
+    ///  .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn request_reviews(
+        &self,
+        pr: u64,
+        reviewers: Vec<String>,
```

Comment:
Why not use `impl Into Vec<String>`? This should allow to drop the `vec!` above.